docs/libcurl/opts/CURLOPT_CURLU.3

```diff
@@ -29,7 +29,7 @@ CURLOPT_CURLU \- set URL with CURLU *
 CURLcode curl_easy_setopt(CURL *handle, CURLOPT_CURLU, void *pointer);
 .SH DESCRIPTION
 Pass in a pointer to the \fIURL\fP to work with. The parameter should be a
-CURLU *.
+CURLU *. Setting CURLOPT_CURLU will explicitly override CURLOPT_URL.
```

Comment:
if you write them as `\fICURLOPT_CURLU(3)\fP` and `\fICURLOPT_URL(3)\fP` they will appear suitably highlighted in man pages and hyperlinked on the web pages.

Reply:
ack